pubsub/iterator.go

```diff
@@ -893,7 +893,7 @@ func (it *messageIterator) pingStream() {
 	spr := &pb.StreamingPullRequest{}
 	it.eoMu.RLock()
 	if it.sendNewAckDeadline {
-		spr.StreamAckDeadlineSeconds = int32(it.ackDeadline())
+		spr.StreamAckDeadlineSeconds = int32(it.ackDeadline()) / int32(time.Second)
```

Comment:
Might be more readable to use https://pkg.go.dev/time#Duration.Seconds and cast from there, rather than doing unit division.